tests/ast/data/session_range.test

```diff
@@ -0,0 +1,23 @@
+## TEST CASE
+
+r = session.range(10)
+
+r = session.range(1, 10)
+
+r = session.range(1, 10, 2)
+
+r = session.range(1, None)
+
+r = session.range(1, None, 2)
+
+## EXPECTED OUTPUT
+
+res1 = session.range(start=10, stop=None, step=1)
```

Comment:
We probably shouldn't generate kwargs in the unparser in this case.

Reply:
I merged the corresponding PR in the other codebase, should I remove the kwargs from there?

Reply:
Yep.

Reply:
Here's the PR for that: https://github.com/snowflakedb/snowflake/pull/198094